events/wc_culture_conversion_events.txt

```diff
@@ -0,0 +1,185 @@
+
+# ###########################################
+# # Warcraft culture conversion events   	#
+# ###########################################
+
+namespace = WCMLT
+
+# ###########################################
+# # Province conversions                    #
+# ###########################################
+
+# Human province in Barrens/Dustwallow converts to Theramorean
+province_event = {
+	id = WCMLT.1
+	desc = EVTDESC_WCMLT_1
+	picture = GFX_evt_lordaeron_coast
+
+	trigger = {
+		should_evolve_cultures_trigger = yes
+		year = 600
+		culture_group = human_group
+		NOT = { culture = theramore }
```

Comment:
Since there's more than one "colonial" culture it makes sense to add them all here. Otherwise, it'll be impossible to convert these lands as ruler of other "colonial" culture. I would add colonial_culture_trigger scripted trigger.
Here's article about what it's and how to do it...
https://ck2.paradoxwikis.com/Scripting#Scripted_block